protractor_tests/question.spec.js

```diff
@@ -16,19 +16,26 @@
  * @fileoverview End-to-end tests for loading all pages.
  */
 
-var QuestionsPage = require('./questions.pageObject.js');
-var utils = require('./utils.js');
+var QuestionsPage = browser.params.questionsPage;
+var utils = browser.params.utils;
 
 describe('submitting questions', function() {
+
   it('should successfully submit wrong code to a task', function() {
     var questionsPage = new QuestionsPage();
 
+    browser.waitForAngularEnabled(false);
```

Comment:
This is a lot of boilerplate to repeat for every test. I wonder whether it makes sense for the six lines to go into a utils.setUpQuestionPage() method?

Reply:
I'm going to put it in questionsPage.setUp(), instead.